cubedash/index/postgis/_api.py

```diff
@@ -376,39 +355,65 @@ def collections_search_query(
         bbox: tuple[float, float, float, float] | None = None,
         time: tuple[datetime, datetime] | None = None,
         q: Sequence[str] | None = None,
-    ) -> Result:
-        collection = self.collection_cols().subquery()
-        query = select(collection).where(collection.c.period_type == "all")
-
+    ) -> list[Row]:
+        # STAC Collections only hold a bounding box in EPSG:4326, no geometry
+        # Calculate the bounding box on the server, it's far more efficient
+        collection_bbox = func.Box2D(
+            func.ST_Transform(TimeOverview.footprint_geometry, 4326)
+        )
+        bbox_array = array(
+            [
+                func.ST_XMin(collection_bbox),
```

Comment:
Not knowing Postgis, this looks really strange to me, isn't there some function that gets all 4 corners of a 2D box?

Reply:
It looked really strange to me too, but best I can tell... no, there isn't, this is what everyone does.

The only other way is passing the WKT strings back to Python and parsing the floats out, which, for 4 floats in a box would be totally fine, but doesn't _feel_ totally fine, so I did it this way.